backend_plugin/src/main/evaluators/BlockCommentEvaluator.java

```diff
@@ -103,4 +88,30 @@ public boolean evaluate(DocumentEvent event) {
 
 		return false;
 	}
+
+	/**
+	 * Helper function that determines if the DocumentEvent(s) provided is a double "//" inputted from the user.
+	 * This will catch both manually typing "/" then "/' as well as using ctrl + / to comment out a single line
+	 * @param event DocumentEvent that occurred
+	 * @return true if a double backslash was placed on a single line, false otherwise
+	 */
+	private boolean doubleBackSlash(DocumentEvent event) {
+		currentOffset = event.getOffset();
+		currentInsert = event.getText();
+
+		// Currently checking for consecutive ctrl + / has limitations with Eclipse API
+		// Check if a "//" was added through copy/paste or ctrl + /
+		// if (event.getText().equals(DOUBLE_SLASH)) {
+		//	 return true;
+		// }
+
+		// Check if there was a single "/" typed followed by another "/"
+		if (prevOffset == currentOffset - 1) {
```

Comment:
Not sure if we care about this but this doesn't detect if a user types `/` then goes before the one they just typed and types another one.

Reply:
This doesn't catch quite a few things. We should discuss and see what we can do to catch more edge cases